Makefile

```diff
@@ -48,7 +48,7 @@ integration: generate-certs
 	@echo "** RUNNING INTEGRATION TESTS **"
 	./test/integration/pretest.sh
 	docker-compose -p app -f docker-compose.yml build
-	docker-compose -p app -f docker-compose.yml up -d --remove-orphans
+	docker-compose -p app -f docker-compose.yml up --remove-orphans --wait --wait-timeout 90
```

Comment:
I would keep the `-d` in there just to make it obvious that you're wanting it to be detached, but want to wait for it to be ready. It being implied makes it less obvious.